src/MapView/utils.ts

```diff
@@ -0,0 +1,12 @@
+// ESLint rule forces the usage of default export if there is only one export.
+// But not using default export here as more util functions might be added later.
+// eslint-disable-next-line import/prefer-default-export
+export const getBounds = (waypoints: Array<[number, number]>): {southWest: [number, number]; northEast: [number, number]} => {
```

Comment:
Let’s try to keep the style consistent as the App repo - so making this a function

and then export default {
Function
}